packages/core-delivery/plugins/send-mail.js

```diff
@@ -25,11 +25,13 @@ class SendMail extends DeliveryAdapter {
     return type === 'SHIPPING';
   }
 
-  isActive() { // eslint-disable-line
+  isActive() {
+    // eslint-disable-line
```

Comment:
This will lead to more warnings I guess :)

Reply:
not sure why the reformat ;-)

Reply:
prettier but shittier 🤟